templates/definition/meter/huawei-sun2000.yaml

```diff
@@ -8,19 +8,45 @@ capabilities: ["battery-control"]
+
+      Die Verbindung zum Wechselrichter erfolgt über Modbus. Die folgenden Details sollten Ihnen bei der Einrichtung der Verbindung helfen.
+
+      - Die Modbus-ID sollte 0 sein, es sei denn, Sie verwenden eine RS485-Verbindung.
+      - Die IP-Adresse sollte auf die IP-Adresse des Wechselrichters oder auf 192.168.200.1 gesetzt sein, wenn Sie eine Verbindung zum Wechselrichter-Zugangspunkt herstellen.
+      - Der Port sollte auf 6607 eingestellt sein, wenn Sie eine Firmware-Version ab Dezember 2021 verwenden, andernfalls sollte er auf 502 eingestellt sein.
+
+      Sie können eine der folgenden Optionen verwenden, um eine Verbindung zum Wechselrichter herzustellen:
+      - Sie können Modbus/TCP in den Kommunikationseinstellungen des Wechselrichters aktivieren. Dazu benötigen Sie ein Installateurkonto.
       Siehe https://forum.huawei.com/enterprise/en/modbus-tcp-guide/thread/667250677153415168-667213868771979264
+      - Sie können sich direkt mit dem Zugangspunkt des Wechselrichters verbinden (SUN2000-<Seriennummer> wifi). Dies ist derselbe Zugang, der für die
+      Geräteinbetriebnahme verwendet und erfordert keinen besonderen Zugriff. Wenn diese Option verwendet wird, sollte der Host auf
+      192.168.200.1 gesetzt werden, die Standard-IP-Adresse des Wechselrichters.
+      Siehe https://github.com/wlcrs/huawei_solar/wiki/Connecting-to-the-inverter#connect-to-the-inverter-ap-sun2000-serial_no-wifi
     en: |
       Grid and Battery require the PowerSensor.
-      Modbus/TCP requires activation using "maintenance access" within the communication settings of the inverter.
+
+      The connection to the inverter is done via Modbus. The details below should help you to set up the connection.
+
+      - Modbus ID should be 0, unless you're connecting using RS485.
+      - IP address should be set to the IP address of the inverter or 192.168.200.1 if you're connecting to the inverter access point.
+      - Port should be set to 6607 if you're using a firmware version from December 2021 or later, else it should be set to 502.
+
+      You can use one of the following options to connect to the inverter:
+      - You can activate Modbus/TCP within the communication settings of the inverter, this requires an installer account.
       See https://forum.huawei.com/enterprise/en/modbus-tcp-guide/thread/667250677153415168-667213868771979264
+      - You can connect to the inverter access point directly (SUN2000-<serial_no> wifi). This is the same access used for
+      device commissioning and does not require any special access. If this option is used, the host should be set to
+      192.168.200.1 which is the default IP address of the inverter.
+      See https://github.com/wlcrs/huawei_solar/wiki/Connecting-to-the-inverter#connect-to-the-inverter-ap-sun2000-serial_no-wifi
 params:
   - name: usage
     choice: ["grid", "pv", "battery"]
     allinone: true
   - name: modbus
     choice: ["rs485", "tcpip"]
+    port: 6607
```

Comment:
Die meisten dürften doch über den SDongleA gehen und da ist 502 der Standard-Port. 6607 wird nur verwendet, wenn man direkt per WLAN auf den SUN2000 geht. Und das ist IMHO ein Nahfeld-WLAN, das nicht sehr weit reicht und eigentlich nur dafür da ist, den SUN2000 einzustellen und zu warten. Was du dokumentierst, ist daher eher ein Randfall!?

Reply:
The Home Assistant integration advices against connecting using the SDongleA because they found it to be less stable, no experience with that however! 

https://github.com/wlcrs/huawei_solar/wiki/Connecting-to-the-inverter#via-the-ip-of-your-sdongle

As also suggested by @premultiply I'll revert the change to the default config and only keep the documentation if the main way of setting up the connection within evcc is using the SDongleA.

It's true that connecting directly to the SUN2000 is meant for the installer, it does however seem to allow you to have modbus access without an installer account.

Reply:
> The Home Assistant integration advices against connecting using the SDongleA because they found it to be less stable, no experience with that however!

I do have a lot of experience with it and it indeed sucks. I switched to RS485.

> It's true that connecting directly to the SUN2000 is meant for the installer, it does however seem to allow you to have modbus access without an installer account.

I never used it for the Modbus purpose. But AFAIK you can't connect it to a WiFi infrastructure, meaning that it can only be used as ad-hoc network. Suppose you use a Raspberry Pi and connect to the SUN2000 directly via WiFi?

Reply:
Yes indeed, a Raspberry Pi, I use an ethernet cable for internet connection and the wifi module for connecting to the inverter. 

There's an explanation on the wiki on how you can make a connection to the inverter and the internet at the same time. The easiest way is having both a wireless connection (to the inverter) and wired connection (to the internet) though: https://github.com/wlcrs/huawei_solar/wiki/Connecting-to-the-inverter#getting-connectivity-between-ha-on-your-home-network-and-the-inverter-ap